patchwork/app.py

```diff
@@ -40,7 +40,7 @@ def _get_config_path(config: str, patchflow: str) -> tuple[Path | None, Path | N
         ignore_unknown_options=True,
     )
 )
-@click.version_option(message="%(version)s")
+@click.version_option(version='0.0.6',message="%(version)s")
```

Comment:
This will hardcode the version to be `0.0.6`. We want to read the current version from the `pyproject.toml` file. Since we use poetry try if `version=poetry.version` works?

Reply:
I updated the file to use a subprocess to get the file version using a function. I didn't want to edit the cli code for click but if you want I can try to integrate it.

Reply:
@GauranshMathur Can we avoid running a command just to get the version. How about read the `pyproject.toml` directly and get `version = pyproject['tool']['poetry']['version']` ? Something similar to https://github.com/LeoIV/BencherScaffold/blob/3e0afead297f319f705a39cfba49c385f133cc71/get_version.py#L4

Reply:
I have made the changes as requested, please tell me if there is anything else that i should add or change

Reply:
@GauranshMathur as mentioned by @CTY-git in #57 doing `@click.version_option(message="%(version)s", package_name="patchwork-cli")` will fix the issue, as the `package_name` is not correct. That might be the best way to fix the issue. If you can make the change and test it in this PR, I can merge it in. Thanks.

Reply:
I added the changes and tested it on my local machine it should work